### PR TITLE
Add Key Vault release notes

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

**Review by kinelski — CHANGES_REQUESTED** [review]

File: releases/2020-10/dotnet.md [hunk 1]
```diff
@@ -60,6 +67,10 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 
 - `EventData` has been integrated with the new Schema Registry service, via use of the `SchemaRegistryAvroObjectSerializer` with the `BodyAsBinaryData` member.
 
+### Key Vault
```
[the remark this answers is not in the document]
Comment (heaths) [reply releases/2020-10/dotnet.md:70]: There's 4 changelogs, all saying the same thing. Do you want 4 headers for 4 links with content that is exactly the same one-liner?

[the remark this answers is not in the document]
Comment (kinelski) [reply releases/2020-10/dotnet.md:70]: Good point. We can leave it alone.

**Review by kinelski — APPROVED** [review]

**Pull request merged**